resources/ditaotdaykeys/berlin17.ditaval

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<val>
+    <prop action="exclude" />
```

Comment:
Why these `.ditaval` files are necessary eludes me, though I see the unwelcome effects that appear when they're removed, so I appreciate the intention behind the workaround.

That said, is there any particular reason we need 7 copies of the same file with different names? Wouldn't it suffice to refer to a single file (such as `all-conferences.ditaval`) everywhere the workaround is required?

Reply:
At this point, no, there is no need to have the extra files. I had grand plans to use branch filtering and keyscopes to eliminate some files. Under that design having different `.ditaval` files would have been necessary. I no longer plan to pursue that design. Therefore, we can simplify.